### ISSUE TITLE
Are dynamic objects annotated in the data annotation?

### ISSUE THREAD

**Issue by ChenYutongTHU** [facebookresearch/vggt-omega#22]
Dear authors,

Thanks so much for this great great work!

I have a question regarding the data annotation pipeline.
According to the paper (Sec 3.5.2),  only static regions are matched and estimated by SfM.
Therefore, there is no depth annotation for dynamic objects. The model can only learn their reconstruction through labeled datasets, not these SfM-labeled in-the-wild dataset.
Do I understand it correctly?

Thanks so much!

**Comment by jytime** [comment]
Hi, It can be annotated but not guaranteed to be accurate, so we drop the annotations in all of those dynamic areas.

**Comment by ChenYutongTHU** [comment]
Thanks for the reply!

I have another question regarding model's performance on scannetpp.
Compared with the performance gain on other datasets, VGGT-Omega's accuracy on scannetpp somehow regresses.

For example, in this 4-view sequence, VGGT-Omega achieves camera extrinsics AUC@3/5/10=28%/ 53%/ 77%.
with the points/rmse_mean = 22 cm. 

The results are still coarsely aligned but not that accurate compared with Pi3X. Is this because of the training data distribution?

<img width="1219" height="741" alt="Image" src="https://github.com/user-attachments/assets/e100bae4-78f0-475a-aa24-c4c24d888f0d" />

Thanks :)

**Comment by jytime** [comment]
Hi you can check our discussion of the data quality in the appendix section. Basically we think the ground truth of scannet series is not that perfect, and hence we totally removed the whole scannet series from our training data. That's why you may find our model's estimation does not exactly align with scannet's ground truth.
